DataMigration/Output/ElasticSearch/Entity/Attribute/Model/Attribute.cs

```diff
@@ -1,165 +1,126 @@
 ﻿using System.Collections.Generic;
-using System.Linq;
-using DataMigration.Input.Episerver.ContentProperty.Model;
-using DataMigration.Output.ElasticSearch.Entity.Attribute.Helper;
-using Newtonsoft.Json;
+using Nest;
 
 namespace DataMigration.Output.ElasticSearch.Entity.Attribute.Model
 {
-    public class Attribute : Entity
+    //TODO Can we change this name to something that does not get confused with System.Attribute ? :) 
```

Comment:
I think we can to avoid future mistakes. Do you have any idea? ProductAttribute, EntityAttribute?